assets/blocks/quiz/quiz-block/quiz-validation.js

```diff
@@ -61,24 +114,7 @@ const IncompleteQuestionsNotice = ( { count, onClick } ) => (
  * @param {Function} props.setMeta
  */
 const QuizValidationResult = ( { clientId, setMeta } ) => {
-	const incompleteQuestions = useSelect(
```

Comment:
See https://developer.wordpress.org/news/2024/03/28/how-to-work-effectively-with-the-useselect-hook/#be-careful-about-transforming-data-inside-the-callback

We caught this as part of this PR because it was also causing a warning.